### PR TITLE
Generate CrossGen Pdbs as part of bulid

### PR DESCRIPTION
**Pull request by vancem**
This change has two parts.  The first part changes build.cmd and the pkgproj
so that as part of generating a .NET Core package we also generate the PDBS
for the Native System.Private.Corelib.ni.dll.

The second part changes crossgen so that it marks the Native Images as having
a list of 0 exports.   This is important because we want the generated PDBs
to be naturally published on the symbol server, and currently the symbol server
skips DLLs what are 'resource only' (no imports or exports).   Because native
images don't use 'normal' PE imports and exports, they get optimized away
by the symbol server.  Making a empty list of exports fixes this.

@jkotas,  @JohnChen0

### CODE REVIEW

File: src/zap/zapheaders.cpp [hunk 1]
```diff
@@ -288,18 +288,49 @@ void ZapDebugDirectory::Save(ZapWriter * pZapWriter)
...
-
         SaveNGenDebugDirectoryEntry(pZapWriter);
         SaveOriginalDebugDirectoryEntry(pZapWriter);
-
     }
 }
 
+ZapPEExports::ZapPEExports(LPCWSTR dllPath) 
+{
+	m_dllFileName = wcsrchr(dllPath, '\\');
```
Comment (jkotas) [inline src/zap/zapheaders.cpp]: The slash will be other way on Unix.

**Comment by vancem** [comment]
@dotnet-bot  test Ubuntu x64 Formatting 
@dotnet-bot test Windows_NT x64 Formatting

**Comment by vancem** [comment]
@dotnet-bot test Linux ARM Emulator Cross Debug Build

**Comment by vancem** [comment]
@dotnet-bot  test Linux ARM Emulator Cross Debug Build

File: src/zap/zapheaders.cpp (hunk 1 above)
Comment (vancem) [inline src/zap/zapheaders.cpp]: Fixed

**Pull request merged**